CONTRIBUTING.md

```diff
@@ -130,6 +131,26 @@ when creating documentation. Refer to our [Component Guide](COMPONENT_GUIDE.md)
 4. If you are willing to assist in the process of creating a new page, then follow the instructions above to `Fork` or `Clone` the repo and make your edits.
```

Comment:
```suggestion
4. If you are interested in creating a new page, then follow the instructions above to `Fork` or `Clone` the repo and make your edits.
```

Reply:
I like the suggestion, but it's outside the scope of the PR. Mind making a new PR? I suspect that sentence is also used in [newrelic/developer-website](https://github.com/newrelic/developer-website/blob/develop/CONTRIBUTING.md) as well.

Reply:
We prefer contractions. "If you're interested..."